src/libraries/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -5217,11 +5217,23 @@ public abstract partial class X86Base
         internal X86Base() { }
         public static bool IsSupported { get { throw null; } }
         public static (int Eax, int Ebx, int Ecx, int Edx) CpuId(int functionId, int subFunctionId) { throw null; }
+        [System.Runtime.Versioning.RequiresPreviewFeaturesAttribute("DivRem is in preview.")]
+        public static (uint Quotient, uint Remainder) DivRem(uint lower, uint upper, uint divisor) { throw null; }
+        [System.Runtime.Versioning.RequiresPreviewFeaturesAttribute("DivRem is in preview.")]
+        public static (int Quotient, int Remainder) DivRem(uint lower, int upper, int divisor) { throw null; }
+        [System.Runtime.Versioning.RequiresPreviewFeaturesAttribute("DivRem is in preview.")]
+        public static (nuint Quotient, nuint Remainder) DivRem(nuint lower, nuint upper, nuint divisor) { throw null; }
+        [System.Runtime.Versioning.RequiresPreviewFeaturesAttribute("DivRem is in preview.")]
+        public static (nint Quotient, nint Remainder) DivRem(nuint lower, nint upper, nint divisor) { throw null; }
```

Comment:
Have these APIs gone through API review?

Reply:
Yes: https://github.com/dotnet/runtime/issues/27292#issuecomment-849856474

Reply:
I assume that you are asking about this specific overload - I do not see it in the approved set.

Reply:
The general premise of extending the intrinsics to support `nint/nuint` was reviewed and approved already: https://github.com/dotnet/runtime/issues/52021

AFAIR, we covered at the time in API review that any other already approved but NYI APIs were also covered by that.

Reply:
The same general approval for `nint/nuint` for Arm64 was done here: https://github.com/dotnet/runtime/issues/52027

Reply:
Indeed, it's covered in the approval explicitly: https://github.com/dotnet/runtime/issues/27292#issuecomment-849856474